data/json/items/gunmod/laser_gunmods.json

```diff
@@ -99,5 +100,97 @@
     "damage_modifier": { "damage_type": "heat", "amount": 10 },
     "energy_drain_multiplier": 2.0,
     "min_skills": [ [ "weapon", 4 ], [ "electronics", 5 ] ]
+  },
+  {
+    "id": "add_intake_fan",
+    "type": "GUNMOD",
+    "name": { "str": "additional intake fan" },
+    "description": "An additional fan that mounts to the radiator intake. It slightly increases the cooling performance of the gun.",
+    "weight": "600 g",
+    "volume": "270 ml",
+    "integral_volume": "150 ml",
+    "price": 84000,
+    "price_postapoc": 1000,
+    "install_time": "20 s",
+    "material": [ "steel", "plastic" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "radiator",
+    "mod_targets": [ "pistol", "rifle" ],
+    "cooling_value_modifier": 0.5
+  },
+  {
+    "id": "heat_dissipation_sheets",
+    "type": "GUNMOD",
+    "name": { "str": "heat dissipation sheets" },
+    "description": "A micro-textured sheet that is applied to the heat exchange portion of the radiator. It increases cooling performance in a format that depends on the gun's original cooling performance.",
+    "weight": "150 g",
+    "volume": "600 ml",
+    "integral_volume": "0 ml",
+    "price": 84000,
+    "price_postapoc": 1000,
+    "install_time": "7 m",
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "radiator",
+    "mod_targets": [ "pistol", "rifle" ],
+    "cooling_value_multiplier": 1.5
+  },
+  {
+    "id": "add_heat_sink_for_small",
+    "type": "GUNMOD",
+    "name": { "str": "additional heat sink for small" },
+    "description": "This is a common type additional heat sink in the form of thin metal plates are evenly spaced. It increases the heat capacity. Although it is designed for small guns, it does not mean that it cannot be attached to other guns.",
+    "weight": "300 g",
+    "volume": "200 ml",
+    "integral_volume": "0 ml",
+    "price": 84000,
+    "price_postapoc": 1000,
+    "install_time": "5 m",
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "heatsink",
+    "mod_targets": [ "pistol", "rifle" ],
+    "overheat_threshold_modifier": 50
+  },
+  {
+    "id": "thermal_equilibrium_gel_sheet",
+    "type": "GUNMOD",
+    "name": { "str": "thermal equilibrium gel sheet" },
```

Comment:
Also seems like too much sci-fi for vanilla

Reply:
This is completely science fiction.
There are plenty of gel sheets themselves, but the materials that can be used with gels are resins and the like.
Resin melts at about 500°C even with the best technology in reality, so a laser gun that rises to the temperature at which metal melts is not enough at all.
However, I could not think of a logical explanation for the parts that change overheat_threshold by a percentage.

Heatsink frame part "a metal part that bridges the gun's original heat sink"
is better?